src/qt/sendcoinsdialog.cpp

```diff
@@ -122,7 +123,7 @@ void SendCoinsDialog::setClientModel(ClientModel *_clientModel)
     this->clientModel = _clientModel;
 
     if (_clientModel) {
-        connect(_clientModel, SIGNAL(numBlocksChanged(int,QDateTime,double,bool)), this, SLOT(updateSmartFeeLabel()));
+        connect(_clientModel, SIGNAL(numBlocksChanged(int,QDateTime,double,bool)), this, SLOT(updateFeeLabel()));
```

Comment:
Is this still the correct signal?

Reply:
I've left it in despite not using smart fees, just in case I'd missed a scenario where it should run. Seemed safer.